.github/workflows/cd-base.yml

```diff
@@ -94,17 +95,28 @@ jobs:
           IFS='.' read -ra VERSION_PARTS <<< "$CURRENT_VERSION"
           PATCH_VERSION=$((VERSION_PARTS[2] + 1))
           NEW_VERSION="${VERSION_PARTS[0]}.${VERSION_PARTS[1]}.$PATCH_VERSION"
-          sed -i "s/version = \"${CURRENT_VERSION}\"/version = \"${NEW_VERSION}\"/" ./base/Ballerina.toml
+          sed -i "0,/version = \"${CURRENT_VERSION}\"/s//version = \"${NEW_VERSION}\"/" ./base/Ballerina.toml
           echo "NEW_VERSION=${NEW_VERSION}" >> $GITHUB_OUTPUT
+          echo "CURRENT_VERSION=${CURRENT_VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Ballerina Build after incrementing version
+        if: ${{ github.event.inputs.bal_central_environment == 'PROD' }}
+        uses: ballerina-platform/ballerina-action@2201.6.0
+        with:
+          args:
+            pack
+        env:
+          WORKING_DIR: ./base
+          JAVA_HOME: /usr/lib/jvm/default-jvm
 
       - name: Commit changes and make a PR
         if: ${{ github.event.inputs.bal_central_environment == 'PROD' }}
         run: |
           # Commit changes
           git config --global user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
           git config --global user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
-          git add ./base/Ballerina.toml
-          git commit -m "[Automated] Prepare hl7v2 Base for next dev cycle"
+          git add .
```

Comment:
Shall we explicitly add the files without adding all?

Reply:
The Bellerina.toml and Dependancies.toml?

Reply:
yes

Reply:
Updated